tests/unit/test-early-renewal-functions.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class Test_Early_Renewal_Functions extends WP_UnitTestCase {
```

Comment:
Early renewals are not a feature in Subscriptions Core so we don't need unit tests for this.

The function is being tested here (`wcs_get_last_non_early_renewal_order()`) is part of `wcs-renewal-functions.php` and doesn't look like it's used by Core at all, but that's okay.

Because this function is in `wcs-renewal-functions.php`, this unit test should live in `test-wcs-renewal-functions.php` class.